Bottom: 6bb785905ba1e71bca88531257ae47bf009d8d5c
Top:    d2254c7a7ec7a686fe37a5eeb855740d8e6331b2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-01-12 22:30:36 +0000

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 1abf11e..116d494 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -997,10 +997,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
